### PR TITLE
4354 workaround protobuf issue

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

part of #4354 

### Description
workaround as suggested

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [x] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: runtests.sh [hunk 1]
```diff
@@ -327,7 +327,8 @@ export PYTHONPATH="$homedir:$PYTHONPATH"
 echo "PYTHONPATH: $PYTHONPATH"
 
 # by default do nothing
-cmdPrefix=""
+# cmdPrefix=""
+cmdPrefix="$(PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python) "  # FIXME https://github.com/Project-MONAI/MONAI/issues/4354
```
Comment (wyli) [inline runtests.sh:331]: don't any idea how to set the environment variable before running cmd? this doesn't seem to work @rijobro @ericspod

**Review by Nic-Ma — APPROVED** [review]

File: runtests.sh (hunk 1 above)
Comment (ericspod) [reply runtests.sh]: You're resolved it with the `export` which is what I'd have suggested.

**Pull request merged**